### ISSUE TITLE
Please update documentation on how to use unicode engines and fontspec

### ISSUE THREAD

**Issue by callegar** [gsilano/EuropeCV#34]
It is now common to use Lualatex or xelatex. While both can fall back to operate as the traditional latex engine, there is advantage in using their ability to fully support unicode and to use system (otf, ttf) fonts.

Would be good to have some notes in the documentation on how to best use these engines and fonspec.

**Comment by gsilano** [comment]
Hi! Thanks for raising this issue and suggesting the use of Lualatex or xelatex with their advantages in supporting unicode and system fonts. It would be beneficial to provide documentation on utilizing these engines and fonspec effectively. Unfortunately, I don't have the time to explore this implementation currently.

If you have any ideas or are already working on it, please feel free to share your draft or thoughts. I appreciate your input in enhancing our project.

**Comment by callegar** [comment]
As a matter of fact, everything works well with the unicode engines, once one has discovered a few tricks:

- do the obvious `\usepackage{fontspec}` in the preamble
- load the fonts using `\setmainfont`, `\setsansfont`, `\setmonofont`.
- in the sans font make sure that you define a compact face, e.g.  `\setsansfont{TeX Gyre Heros}[FontFace = {c}{\shapedefault}{TeX Gyre Heros Cn }]` otherwise the option `narrow` won't work.
- have a `\renewcommand{\familydefault}{\sfdefault}` if your main family is a serif font

furthemore you may want to have
```LaTeX
\makeatletter
\def\ecv@lfoot{%
  \footnotesize\textup{\ecv@page~-~\ecv@cvofkey}\\
  \textup{\ecv@footername}
}
\makeatother
```
in the preamble if your main font is serif and you want the page number in sans. This changes the `\textrm` to `\textup` in `\def\ecv@lfoot`.

Documenting this might be useful IMHO.

**Comment by gsilano** [comment]
Absolutely! I totally agree with you, but unfortunately, I'm swamped with tasks right now. However, would you be up for helping out by improving the documentation? You can make your changes in the [europecv.tex](https://github.com/gsilano/EuropeCV/blob/master/europecv.tex) file. Once you're done, just open a pull request with your updates. After that, I'll review everything and merge it into the repository. Eventually, I'll also make sure to propagate those changes on CTAN. Thanks a bunch for your support! 😊
